moped-database/README_seed_data.md

```diff
@@ -0,0 +1,39 @@
+### How to generate new seeds
+
+1. Navigate to `./moped-database`
+2. Start the hasura cluster
+3. Add / edit whatever data
+4. ❗️Warning: adding subprojects creates a circular reference that will cause seeds to fail
+5. With pg_dump installed, run this command in your shell. You can add additional tables as needed - do not include lookup tables - these tables are hydrated through migrations:
+
+```shell
+pg_dump --column-inserts --data-only --schema public \
+  -t moped_proj_components \
+  -t moped_proj_components_subcomponents \
+  -t moped_proj_contract \
+  -t moped_proj_entities \
+  -t moped_proj_features \
+  -t moped_proj_financials \
+  -t moped_proj_fiscal_years \
+  -t moped_proj_funding \
+  -t moped_proj_milestones \
+  -t moped_proj_notes \
+  -t moped_proj_partners \
+  -t moped_proj_personnel \
+  -t moped_proj_personnel_roles \
+  -t moped_proj_phases \
+  -t moped_proj_tags \
+  -t moped_project \
+  -t moped_project_files \
+  -t moped_user_followed_projects \
+  -t moped_users \
```

Comment:
it just occurred to me that this is a list of all tables we would ideally be tracking in the activity log (except for `moped_users`). cc @chiaberry